crates/shielded_token/src/vp.rs

```diff
@@ -44,16 +45,35 @@ pub struct MaspVp<'ctx, CTX, Params, Gov, Ibc, TransToken, Transfer> {
 }
 
 // Balances changed by a transaction
-#[derive(Default, Debug, Clone)]
+#[derive(Debug, Clone)]
 struct ChangedBalances {
-    // Maps asset types to their decodings
-    tokens: BTreeMap<AssetType, (Address, token::Denomination, MaspDigitPos)>,
+    // Maps unepoched asset types to their decodings
+    unepoched_tokens:
```

Comment:
It would be nice to either use `undated` or `unepoched` consistently in var and field names

Reply:
I would go with `undated`/`dated`, since that seems like the most consistently used terminology